src/typography/src/Paragraph.tsx

```diff
@@ -10,19 +10,19 @@ export interface ParagraphProps extends React.ComponentProps<typeof Box> {
   /**
    * The color (alias or valid color) applied to the text
    */
-  color?: string
+  color: string
 
   /**
    * Font family.
    * Can be: `ui`, `display` or `mono` or a custom font family.
    */
-  fontFamily?: FontFamily
```

Comment:
These aren't really required are they? Can't you pass `undefined` to them and still get back the right values?

Reply:
Some of the theme functions don't handle undefined cases. But these have `defaultProps` so you don't need to set them.